runners/direct-java/build.gradle

```diff
@@ -75,6 +76,8 @@ dependencies {
   needsRunner project(path: ":beam-sdks-java-core", configuration: "shadowTest")
   needsRunner project(path: project.path, configuration: "shadow")
   needsRunner project(path: project.path, configuration: "shadowTest")
+  validatesRunner project(path: ":beam-sdks-java-core", configuration: "shadowTest")
```

Comment:
This will duplicate them, I think. Did you confirm? Because ValidatesRunner extends NeedsRunner, all the VR tests should be pulled in by the other execution. I think your version is cleaner, so I would agree to break this relationship. We could do NR minus VR for the needsRunner gradle suite, or just solve it with documentation/rename.

Reply:
Added a comment; the intention is to have a DirectRunner version of the `@ValidatesRunner` suite so that I can run the same set of tests but in a faster in-memory runner. It's not being used for Jenkins.